contrib/postprocessing/example.py

```diff
@@ -103,7 +98,4 @@
 # using the same structure as in PyVista documentation
 # (https://docs.pyvista.org/)
 
-# To write those, the following can be used:
-example.write_vtu()

```

Comment:
So the write_vtu is now done automatically?

Reply:
Yes, exactly. It is done on the flight